python/pyarrow/io.pxi

```diff
@@ -1020,6 +1078,22 @@ cdef class OSFile(NativeFile):
 cdef class FixedSizeBufferWriter(NativeFile):
     """
     A stream writing to a Arrow buffer.
+
+    Examples
+    --------
+    >>> import pyarrow as pa
+    >>> buf = pa.allocate_buffer(5)
+    >>> bit = b'abcde'
+    >>> writer = pa.FixedSizeBufferWriter(buf)
+    >>> writer.write(bit)
```

Comment:
```suggestion
    >>> writer = pa.FixedSizeBufferWriter(buf)
    >>> writer.write(b'abcde')
```

(since `bit` is not used elsewhere, passing it directly avoids possible confusion with variable names)